### PR TITLE
Increase availibility of combat knives to security

### PR DESCRIPTION
**Pull request by Ghagliiarghii**
## About the PR

Added combat knives to sec vend and added combat knife to the sec lathe for a cost of 2.5 steel and 1 plastic (the handle has a plastic grip, but don't worry, it's still very rugged.)
## Why / Balance

There's a recent bug where sometimes people start their shifts with no shoes. This doesn't directly address that, however, all security start with a knife in their boots as a sidearm (so it's clearly not a restricted item), but there is currently no good way to obtain more of these knives which didn't make sense to me personally and several users on a discord discussion ("Latejoined as HoS with no shoes") seemed to agree.
It downright feels weird to, as a security officer, ask Chef for a kitchen knife for my cutting & slashing needs or go the rest of the shift with a glass shiv or something.
There's also the matter of Diona who work for Security not able to wear shoes, and so who don't start with shoes, and so don't have easy access to a tool often used by security officers and so are unfairly under-equip to do their jobs.

**Changelog**

:cl:

- tweak: Security can now find replacements for their trusty Combat Knife in the SecVend, or craft them with the Sec Lathe.

### CODE REVIEW

**Comment by deltanedas** [comment]
bug or not diona have no shoes so this is very needed for diona sec

**Comment by Ghagliiarghii** [comment]
(reasoning on whitelist is I feel it's related, it's useful for diona who can't have shoes or others who want to wear a different style of shoes, I don't think security webbing have the whitelist at all and that's a roundstart item now with loadouts and previously with secvend, we've been talking about completely removing the whitelist from belts but obviously that's outside of scope, and many real-life examples of sheaths for knives on belts in real life and in real security / police / military applications for verisimilitude (believability, intuition for what can go in this belt for a new player) and a different role (chef) already has knives whitelisted on their belt for some precedence)

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     ClothingNeckScarfStripedRed: 3
     ClothingOuterArmorBasicSlim: 2 
     ClothingEyesBlindfold: 1
-    ClothingShoesBootsCombat: 1
+    ClothingShoesBootsCombatFilled: 1
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml:24]: This change is unnecessary please revert.

Comment (Ghagliiarghii) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml:24]: I'm least attached to this part of it, sure no problem

**Pull request merged**